dotcom-rendering/playwright/lib/iframe.ts

```diff
@@ -0,0 +1,13 @@
+import type { Locator, Page } from '@playwright/test';
+import { expect } from '@playwright/test';
+
+const getIframeBody = async (
+	page: Page,
+	iframeSelector: string,
+): Promise<Locator> => {
+	const iframeBodyLocator = page.frameLocator(iframeSelector).locator('body');
+	await expect(iframeBodyLocator).toBeAttached({ timeout: 10000 });
```

Comment:
Might be more readable with a separator:

```suggestion
	await expect(iframeBodyLocator).toBeAttached({ timeout: 10_000 });
```